tests/lvmo/test_lvm_manual_diskpath.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import os
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.templating import load_yaml, dump_data_to_temp_yaml
+from ocs_ci.utility.lvmo_utils import (
+    delete_lvm_cluster,
+    get_blockdevices,
+    get_disks_by_path,
+    lvmo_health_check,
+)
+from ocs_ci.framework.pytest_customization.marks import skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version
+from ocs_ci.ocs.ocp import OCP
+
+
+log = logging.getLogger(__name__)
+
+
+LVMCLUSTER_TEMPLATE = os.path.join(
+    constants.TEMPLATE_DEPLOYMENT_DIR_LVMO, "lvm-cluster-default.yaml"
+)
+
+
+def create_lvm_cluster_cr_with_device_selector(disks):
+    """
+    Creates LVMCluster configuration file with deviceSelector and disks list
+
+    Args:
+        disks (list): list of disks (str) that selected for lvm-cluster
+
+    """
+    lvm_cr_dict = load_yaml(LVMCLUSTER_TEMPLATE)
+    lvm_cr_dict["spec"]["storage"]["deviceClasses"][0]["deviceSelector"] = {
+        "paths": disks
+    }
+    tmp_file_path = "/tmp/lvm_cr.yaml"
```

Comment:
You should probably use tempfile.NamedTemporaryFile module here.
BTW is this test suppose work if executed multiple times on the same cluster?

Reply:
it should run multiple time on same cluster,yes. im about to add pytest.parametrize